elastic/kibana
# [ResponseOps][Alerts] Fix stack alerts page table loading twice when filters load

## Summary

Waits for the filter controls bar to load before triggering the alerts table loading.

Fixes #183412

## To verify

1. Open the Network DevTools tab, filter by `Fetch/XHR` and search `bsearch`
2. Navigate to the Stack alerts page (`Stack Management` > `Alerts`)
3. Verify visually that the table loads only once
4. Verify that just one `bsearch` network request is performed
